app/assets/javascripts/spotlight/application.js

```diff
@@ -24,5 +24,6 @@
 //= require bootstrap-tagsinput
 //= require jquery.serializejson
 //= require clipboard
+//= require dragula.min
```

Comment:
dragula is really nice and simple! however it doesn't allow for nesting objects so if we were to implement it into spotlight to replace nestable.js we would have to come up with a new way to do that for feature pages. might be best to stick with nestable and ill have to change my code at some point so that it uses that in stead.